scripts/sentiment_analysis/textCNN.py

```diff
@@ -0,0 +1,55 @@
+"""textCNN model."""
+
+import mxnet as mx
+from mxnet import gluon
+from mxnet.gluon import HybridBlock
+import gluonnlp as nlp
```

Comment:
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

You should put a blank line between each group of imports. Sort imports alphabetically within each group.